app/jobs/send_webhook_job.rb

```diff
@@ -8,6 +8,7 @@ class SendWebhookJob < ApplicationJob
   retry_on ActiveJob::DeserializationError, wait: :exponentially_longer, attempts: 6
 
   WEBHOOK_SERVICES = {
+    'alert.subscription_to_be_terminated' => Webhooks::Subscriptions::TerminatingSoonService,
```

Comment:
The webhook should be scoped to the resource type it is alerting about, here `subscription`.

What about something like `subscription.reaching_termination`.

In addition, I think we are doing ourself a favor by using the same name for both the webhook service and the event name :). So here, `Webhooks::Subscriptions::ReachingTerminationService`